docs/products/redis/get-started.rst

```diff
@@ -1,59 +1,35 @@
-Get Started with Aiven for Redis
+Get started with Aiven for Redis
 ================================
 
-Get up and running with Aiven for Redis. This article shows you how to set up a Redis service in Aiven and learn more about the platform.
+Aiven for Redis services are managed from the `Aiven
+Console <https://console.aiven.io/>`__. Create an Aiven for Redis service by following the steps laid out in :doc:`this article </docs/platform/howto/create_new_service>`.
 
-Create a new Aiven service
---------------------------
+Once the service creation is initiated, you can go to its dedicated service page where the new service is shown
+with an indicator that it is being created.
 
-To create a new Aiven for Redis service:
+Click the service name in the list to go to the "**Overview**" page.
```

Comment:
Because of the previous sentence we're already in the service page, so we don't have a list, right?
You can replace it with something like "The **Overview** tab shows connection...." and continue with your paragraph :)